library/platform_util.c

```diff
@@ -28,10 +28,13 @@
 
 #include "mbedtls/platform_util.h"
 
-#include <stddef.h>
+#ifndef mbedtls_platform_zeroize
+#ifndef MBEDTLS_PLATFORM_ZEROIZE_ALT
```

Comment:
In Mbed TLS it is prefered to use `#if !defined(MACRO)` rather than `#ifndef MACRO`. This would need to be changed here as well as in other places.

Reply:
Quick check of the current `development` branch showed nearly one hundred of files where `#ifndef` was used. That is why I thought it was still legal to use shorter and simpler directive.
No objections to change it according to the latest style if needed.

Reply:
Do note, that except for one case, which may require investigation, these are all used for the header guards. Therefore, please change the directive.

Reply:
The directives have been changed.
Thanks for reviewing.